packages/kbn-dev-utils/src/proc_runner/proc_runner.ts

```diff
@@ -99,7 +99,7 @@ export class ProcRunner {
         // wait for process to log matching line
         await proc.lines$
```

Comment:
Feels like we should fix `proc.lines$` to properly be an `Observable<string>` rather than explicitly typing `string` in the filter()